SleepEngine/SleepEngine.vcxproj

```diff
@@ -22,7 +22,7 @@
     <VCProjectVersion>15.0</VCProjectVersion>
     <ProjectGuid>{DE4774CF-D98B-4D53-B16E-875A7CCD77B1}</ProjectGuid>
     <RootNamespace>SleepEngine</RootNamespace>
-    <WindowsTargetPlatformVersion>10.0.17763.0</WindowsTargetPlatformVersion>
+    <WindowsTargetPlatformVersion>10.0.17134.0</WindowsTargetPlatformVersion>

```

Comment:
Stop changing WinAPI versions in every merge)

Reply:
then don't change them back - i don't have any othe installed)